roles/kvirt_vm/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Run validations
+  ansible.builtin.include_tasks: validations.yml
+
```

Comment:
I think the deletion was here but now is gone, should it be here?

Reply:
It was, but no access to the `vm.name` at this point